astronomer-starship/starship/main.py

```diff
@@ -15,6 +18,18 @@
 
 from python_graphql_client import GraphqlClient
 
+from starship.services.aws.secrets_manager import (
+    SecretsManagerHook as AwsSecretsManagerHook,
+)
+from starship.services.aws.systems_manager import (
+    SystemsManagerHook as AwsSystemsManagerHook,
+)
+from starship.services.azure.key_vault import AzureKeyVaultHook
+from starship.services.google.secrets_manager import (
+    SecretsManagerHook as GoogleSecretsManagerHook,
+)
```

Comment:
probably wrap these in a try/except. No need to force people to install azure, etc if they haven't already and I think currently it'd blow up as-written on the import